packages/@okta/i18n/src/properties/login.properties

```diff
@@ -819,6 +819,13 @@ oie.verify.google_authenticator.otp.title = Verify with Google Authenticator
 oie.verify.google_authenticator.otp.description = Enter the temporary code generated in your Google Authenticator app
 oie.google_authenticator.otp.enterCodeText = Enter code
 
+## Custom OTP
+oie.verify.custom_otp.title = Verify with {0}
+oie.verify.custom_otp.description = Enter the authenticator set up code included with the device or provided by your admin.
+oie.custom_otp.description = Enter a temporary code generated from an authenticator device.
+oie.custom_otp.authenticator.default.vendorName = Custom OTP Authenticator
+oie.custom_otp.verify.passcode.label = Enter code
```

Comment:
possible to use the v1 key instead?

Reply:
I added this based on your previous comment for customization capabilities. I think we should clean up all these instances in a different PR , because we have already duplicated this a few times.

https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties#L829
https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties#L840

Reply:
tracked https://oktainc.atlassian.net/browse/OKTA-371477

Reply:
Good catch @nikhilvenkatraman-okta 

The idea of transformer is to map the constructed key with something existing and not create new `value` keys

add a this jira as comment so that people don't add new `value` keys for existing strings